tests/detect-itype/test.rules

```diff
@@ -0,0 +1,5 @@
+alert icmp any any -> any any (itype:10; sid:1;)
+alert icmp any any -> any any ((itype:<15; sid:2;)
+alert icmp any any -> any any (itype:>20; sid:3;)
+alert icmp any any -> any any (itype:8<>20; sid:4;)
```

Comment:
Could you try and make the rules with the same itype conditions but such that all of them alert on the pcap?

Reply:
I added these rules based on what I found in the corresponding unit test. 
Should I alter the pcap to match the rule conditions or rules to match the pcap that I got. (The pcap I have right now has `itype 0 and 8`)
What's the better approach in this scenario?

Reply:
I would say that it's better to adjust the rules to the pcap you have, ensuring that all of the rules match on this PR.